src/components/BillingForm.js

```diff
@@ -1,76 +1,112 @@
-import React, { useState } from "react";
-import { FormGroup, FormControl, ControlLabel } from "react-bootstrap";
-import { CardElement, injectStripe } from "react-stripe-elements";
-import LoaderButton from "./LoaderButton";
-import { useFormFields } from "../libs/hooksLib";
-import "./BillingForm.css";
+import React, { useState } from 'react';
+import { CardElement, injectStripe } from 'react-stripe-elements';
+import { TextField, Divider } from '@material-ui/core';
+
+import LoaderButton from './LoaderButton';
+
+import { useFormFields } from '../libs/hooksLib';
+
+function StripeInput(props) {
```

Comment:
What's happening here?

Reply:
Hey @xuatz

I was trying to integrate the `react-stripe-elements` into `material-ui` unfortunately they don't have a clean way of doing it. One of the author of `material-ui` created a `snippet`. `TextField` component of `material ui` have a `InputProps` property which in a way you could override the default component by passing a custom props. It's not actually documented on their web unfortunately.

But they have a topic for it here:
https://github.com/mui-org/material-ui/issues/16037